Automattic/wp-calypso
# Typography: Use npm package import for Recoleta

#### Changes proposed in this Pull Request

* Updates individual imports of Recoleta to use the new `@automattic/typography` package.
* Adds a `brandFont` boolean prop to `FormattedHeader` and adds it to the headings accessible from the sidebar nav/subsections. This prevents the brand font from showing up on all instances of FormattedHeader, only the ones we specify. 
* Also updates Customer Home task cards and section headings, and Earn primary action card heading

**Visuals**

<img width="1121" alt="Screen Shot 2020-06-16 at 11 14 23 AM" src="https://user-images.githubusercontent.com/2124984/84793684-679c4700-afc3-11ea-8b7f-521b9db53bee.png">
<img width="1083" alt="Screen Shot 2020-06-16 at 11 13 59 AM" src="https://user-images.githubusercontent.com/2124984/84793687-6834dd80-afc3-11ea-8ff3-5ba52825136c.png">
<img width="1086" alt="Screen Shot 2020-06-16 at 11 13 37 AM" src="https://user-images.githubusercontent.com/2124984/84793688-6834dd80-afc3-11ea-9e80-1dbec98872d2.png">

Vietnamese example:

<img width="1680" alt="Screen Shot 2020-06-16 at 11 21 50 AM" src="https://user-images.githubusercontent.com/2124984/84793927-a92cf200-afc3-11ea-8704-479b925774bd.png">


#### Testing instructions

* Switch to this PR
* Check out `/earn`, `/home`, and all of the main pages with headings in the sidebar nav.
* All headings should display in Recoleta
* Switch to an unsupported language like Vietnamese -- all headings should use the fallback serif font instead of Recoleta.
